lib/howl/interactions/replacement.moon

```diff
@@ -0,0 +1,385 @@
+-- Copyright 2012-2015 The Howl Developers
+-- License: MIT (see LICENSE.md at the top-level directory of the distribution)
+
+import Matcher, TaskRunner from howl.util
+import app, command, config, interact, timer from howl
+import highlight, ListWidget, NotificationWidget from howl.ui
+
+append = table.insert
+
+highlight.define_default 'replace_strikeout', {
```

Comment:
I think all themes should explicitly define all the basic styles it needs to define, including the light theme, even though it's currently the default.